composer.json

```diff
@@ -36,22 +36,22 @@
         }
     },
     "require": {
-        "php": "~8.1.0 || ~8.2.0",
-        "fig/http-message-util": "^1.1.2",
-        "laminas/laminas-stdlib": "^3.1",
-        "mezzio/mezzio-router": "^3.14",
-        "nikic/fast-route": "^1.2",
-        "psr/container": "^1.0 || ^2.0",
+        "php": "~8.1.0 || ~8.2.0 || ~8.3.0",
+        "fig/http-message-util": "^1.1.5",
+        "laminas/laminas-stdlib": "^3.18",
+        "mezzio/mezzio-router": "^3.17",
+        "nikic/fast-route": "^1.3",
+        "psr/container": "^1.0 || ^2.0.2",
```

Comment:
Please can we just bump `require-dev` here and leave `require` unchanged except for `php`?

Reply:
done